src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+
+import duke.exceptions.DukeException;
+
+import java.io.*;
```

Comment:
When you import your classes, I think it would be better to list them explicitly.
```suggestion
import java.io.IOException;
import ...
```
This adheres to the coding standards, which should be the same for your tP. Additionally, it gives excellent documentation value for the team. I believe for IntelliJ, if 3 or more classes are imported from the same package, IntelliJ automatically converts it into a wilcard import, this limit can be toggled or turned off within your IDE settings.